src/util/articleRenderers.ts

```diff
@@ -109,9 +191,10 @@ export function renderMCSArticle(json: any, dump: Dump, articleId: string, artic
     }
     html = html.replace(`__SUB_LEVEL_SECTION_${json.remaining.sections.length}__`, ''); // remove the last subcestion anchor (all other anchor are removed in the forEach)
     return html;
-}
+};
+
 
-function groupAlphabetical(items: Array<{ name: string, url: string }>) {
+const groupAlphabetical = (items: Array<{ name: string, url: string }>) => {
```

Comment:
In this PR, there are some functions which are changed from normal function to arrow functions. Is there a particular reason why it is done? IMK, simple functions are faster, takes less memory and are good for optimisation.

Reply:
My only reason is code readability.

Regarding to performance: those days, while arrow functions implementation in v8 was on early experimental stages, they could be significantly slower. Nowadays, it's not that simple. 

Anyways, I tested our own case with the following [benchmark](https://www.typescriptlang.org/play/#code/MYewdgzgLgBGMF4YEYAM7UG4CwAoPoksAJgIZSmIwDaeMMA3mKQLYCmAXDAOQBmIIAEzcANDACuAJwA2XbgAsoUAA7cAviLqNm7OfxDJREmXMUqI6zbnpNWnHvoDMRqbJ7jiqjVtu6eAI1JJNBcTHihgLysbHXtuQMkAVlC3bgBLYBYovABdHHwCgHpCmCDJEAB3AnBoGF4wAEFJcoqqAAo0qDYWCC4myVIATwAeBjg7LmhJNLAAczFXSahpuZg1AD4ASkR1xi1CWtny8WUIBulleVJ-NigM0mkqTu6IADpJNmJxYDY2ttJgMAuKQwIMxM8WNsELsGFp6AdYEcQCcAJLEJ5dFivWLUVA5V5QEAAGRAwAebAAqsplGxJABhUgQX6bfL0egA4DUJGo4g5docrnHZRovkAH1FNBym1ehDJUA6mJZcJgHygUngHNZazEDDUSoK9FV6pgAHl-AArNjAKCvADWbEGEDa3NO50u11u92km2VrwgIEk8p91jZrxYpGUf2kty6kihMOVhtuxthIbZbLuUGjXGjSlp0XTbIhvRgLrOFyuNzuZOk1FzsZyBfTai19D1+RbeDwxTgAfD0jwvHEYGtaXAdTAADk+w8FS8+s0hqNxn4pjN5sY3GvVhttqn4TVEULy+6q16MS93p9vr9-oCOCCwTAIfG9mmYAjS0K0ResTi8QSxKkuSVI0vSjLMsqAouiK-KAoKyLCryMDipK0qyuQc6QsqRqSBqgJaBojB6l2Ia4fAZqWtadoOk6ZZupWno1sG6Z+gGQa+uGkZtPWtKvvuhbkW+hYZp02YwLxkhNoWxZcPRFYetWDx1jGtKNomazKiRuBqKRPZgDOjwVAM1K0tURC9pI-btPx+yHjAAAeVAALLkPI7wgsQIAsG0+oHpAIDRgSaTsG03AGVZDzcH5dQBjAPG3M+VBYElwxwJgz4ANSZXuyr1NOkXSG0ZAUDFukhgcgVsMF7AAKJgMQYURf20Udvk3YlGUlQwMZEZgeZtRda0SC+TswkfvZTlIK5UDuQMDXeb5WqVUFdyhdwQ2tVo-CSPFuZJUgKVpDAaVgBlaTZbl771P0lTFeQpBlXZAWrSFbD1Y1G3NJUW06e1uA9pIw6DnFCWwMdh3nSdKBHZd43NbOMVDUteDlUAA).

Here is some results with 5 elements and 10000 runs:

Normal first
![image](https://user-images.githubusercontent.com/6795530/80574580-cb19e780-8a0a-11ea-97c8-8d1cfee32f53.png)

Arrow first
![image](https://user-images.githubusercontent.com/6795530/80574655-ec7ad380-8a0a-11ea-8c81-1b17bdee9909.png)
As we could see, the variance is within the margin of error in both direction (differences between first two runs and the following ones is V8 optimizations).

Another point of view from V8 developer: https://stackoverflow.com/a/44031830/11372576

Reply:
Thanks for making the comparison and I totally agree that it improves the readability. These are sources I was referring to:
https://www.reddit.com/r/javascript/comments/9f3hhf/bound_function_vs_arrow_function_performance/

But I am happy that in our use-case it is not impacting the performance. :)